rotors_gazebo/models/yosemite/yosemite.sdf

```diff
@@ -8,7 +8,7 @@
           <heightmap>
             <uri>model://yosemite/materials/textures/yosemite.png</uri>
             <size>1000 1000 125</size>
-            <pos>100 0 -30</pos>
+            <pos>0 0 -15</pos>
```

Comment:
With previous coordinates, I noticed what seemed to be a discrepancy between the visual and collison layer of the heightmap - I would sometimes be flying through seemingly free space but the aircraft would crash/stop as if it hit something. Seems to work better with the new coordinates. Plus, as a general practice I think it's clearer when the link origin is aligned with the model origin if possible (at least in the xy-plane in this case).